src/components/ui/coaching-sessions/coaching-notes/toolbar.tsx

```diff
@@ -12,85 +12,109 @@ import {
+          onClick={() => editor.chain().focus().toggleBold().run()}
+          isActive={editor.isActive("bold")}
+          icon={<Bold className="h-4 w-4" />}
+          title="Bold (Ctrl+B)"
+        />
+        <ToolbarButton
+          onClick={() => editor.chain().focus().toggleItalic().run()}
+          isActive={editor.isActive("italic")}
+          icon={<Italic className="h-4 w-4" />}
+          title="Italic (Ctrl+I)"
+        />
+        <ToolbarButton
+          onClick={() => editor.chain().focus().toggleUnderline().run()}
+          isActive={editor.isActive("underline")}
+          icon={<Underline className="h-4 w-4" />}
+          title="Underline (Ctrl+U)"
+        />
+        <ToolbarButton
+          onClick={() => editor.chain().focus().toggleStrike().run()}
+          isActive={editor.isActive("strike")}
+          icon={<Strikethrough className="h-4 w-4" />}
+          title="Strike Through"
+        />
+        <ToolbarButton
+          onClick={() => editor.chain().focus().toggleHighlight().run()}
+          isActive={editor.isActive("highlight")}
+          icon={<Highlighter className="h-4 w-4" />}
+          title="Highlight Text"
+        />
+        <ToolbarButton
+          onClick={() =>
+            editor.chain().focus().toggleHeading({ level: 1 }).run()
+          }
+          isActive={editor.isActive("heading", { level: 1 })}
+          icon={<Heading1 className="h-4 w-4" />}
+          title="Heading 1"
+        />
+        <ToolbarButton
+          onClick={() =>
+            editor.chain().focus().toggleHeading({ level: 2 }).run()
+          }
+          isActive={editor.isActive("heading", { level: 2 })}
+          icon={<Heading2 className="h-4 w-4" />}
+          title="Heading 2"
+        />
+        <ToolbarButton
+          onClick={() =>
+            editor.chain().focus().toggleHeading({ level: 3 }).run()
+          }
+          isActive={editor.isActive("heading", { level: 3 })}
+          icon={<Heading3 className="h-4 w-4" />}
+          title="Heading 3"
+        />
+        <ToolbarButton
+          onClick={() => editor.chain().focus().toggleBulletList().run()}
+          isActive={editor.isActive("bulletList")}
+          icon={<List className="h-4 w-4" />}
+          title="Bullet List"
+        />
+        <ToolbarButton
+          onClick={() => editor.chain().focus().toggleOrderedList().run()}
+          isActive={editor.isActive("orderedList")}
+          icon={<ListOrdered className="h-4 w-4" />}
+          title="Ordered List"
+        />
+        <ToolbarButton
+          onClick={() => editor.chain().focus().toggleCodeBlock().run()}
+          isActive={editor.isActive("codeBlock")}
+          icon={<Braces className="h-4 w-4" />}
+          title="Code Block"
+        />
+        <ToolbarButton
+          onClick={() => setIsLinkDialogOpen(true)}
+          isActive={editor.isActive("link")}
+          icon={<Link className="h-4 w-4" />}
+          title="Link"
```

Comment:
@zgavin1 Can you dynamically have this text change like so:
* "Insert Link (Ctrl+K)" for when inserting a link
* "Update Link (Ctrl+K)" when one has highlighted a link?